lib/readable_streambuffer.js

```diff
@@ -42,6 +42,9 @@ var ReadableStreamBuffer = module.exports = function(opts) {
     if (sendMore) {
       sendData.timeout = setTimeout(sendData, frequency);
     }
+    else {
+      sendData.timeout = null;
```

Comment:
Ah! I see. Only looked at the diff. Do you think this will have another affect in comparison to https://github.com/samcday/node-stream-buffer/pull/28? Not sure why stream#push returns `undefined` instead of `false`?